.vscode/launch.json

```diff
@@ -0,0 +1,25 @@
+{
+    // Use IntelliSense to learn about possible attributes.
```

Comment:
This looks like not strictly json. VScode can still read these files with these comments?

Reply:
JSON .NET :)